File: backend/Services/Review/Review.Application/DTOs/ReviewDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Reviewing.Application.DTOs
{
    public class ReviewDto
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public int TourId { get; set; }
        public string Title { get; set; }
        public string? Comment { get; set; }
        public int? Difficulty { get; set; }
        public int? Score { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime UpdatedDate { get; set; }
    }
}

File: frontend/src/features/review/types/ReviewDto.ts
export interface ReadReviewDto {
  id: number;
  userId: number;
  tourId: string;
  title: string;
  comment?: string;
  difficulty?: number;
  score?: number;
  createdDate: string;
  updatedDate: string;
}

export interface CreateReviewDto {
  userId: number;
  tourId: string;
  title: string;
  comment?: string;
  difficulty?: number;
  score?: number;
}
